src/components/Apps/AppItem.jsx

```diff
@@ -99,6 +101,17 @@ export class AppItem extends React.Component {
       // target app is web
       // run switch listener only if the current app is native mobile
       onClick = this.onAppSwitch
+    } else if (isMobile()) {
```

Comment:
What is the difference with the `isMobileApp()` call?

Reply:
`isMobileApp` checks the cordova context aka the native one. 

Here, we just test if it's the mobile web version... Thanks cozy-device-helper and its name :/

Reply:
It should be more explicit on cozy-device-helper, but I won't block this PR for that

Reply:
But hang on, `isMobileAppAvailable ` can only be checked when `isMobile()` is `true`, right? So maybe we could make this triple condition a bit clearer?

Reply:
In fact the three tests are: 
- if we are on native app and other native app is installed
- else if we are on native app 
- else if we are on mobile 

How can I make them clearer? By renaming the variables?

Reply:
Yeah I think renaming the variables or being more explicit in the commend would help. I know it's not part of this PR but `run switch listener` for example is not very clear.

Something like this I think would help:

```
const isNativeToNative = isMobileAppAvailable
const isNativeToWeb = isMobileApp()
const isWebToNative = isMobile()
```